### PR TITLE
added new fonts

### PR DESCRIPTION
**Pull request by ankushCb**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -64,6 +64,70 @@
...
+@font-face {
+  font-family: "Soleil";
+  font-weight: normal;
+  font-style: normal;
+  src: local("Soleil"),
+  local("Soleil-Regular"),
+  url("./fonts/soleil.otf") format("opentype");
+}
+@font-face {
+  font-family: "Soleil";
```
Comment (Kameshwaran) [inline src/index.css]: It should have different name @ankushCb 
like 'SoleilItalic'

File: src/index.css [hunk 2]
```diff
@@ -64,6 +64,70 @@
   local("Serverless-Regular"),
   url("https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/fonts/serverless-regular-v3.otf") format("opentype");
 }
+@font-face {
+  font-family: "Soleil";
```
Comment (Kameshwaran) [inline src/index.css]: Name should be like 'SoleilRegular'

File: src/index.css [hunk 3]
```diff
@@ -64,6 +64,70 @@
...
+  font-weight: normal;
+  font-style: italic;
+  src: local("Soleil Italic"),
+  local("Soleil-Italic"),
+  url("./fonts/soleil-italic.otf") format("opentype");
+}
+@font-face {
+  font-family: "Soleil";
```
Comment (Kameshwaran) [inline src/index.css]: Name should be `SoleilBoldItalic`

File: src/index.css [hunk 4]
```diff
@@ -64,6 +64,70 @@
...
+  font-weight: 700;
+  font-style: italic;
+  src: local("Soleil BoldItalic"),
+  local("Soleil-BoldItalic"),
+  url("./fonts/soleil-bolditalic.otf") format("opentype");
+}
+@font-face {
+  font-family: "Soleil";
```
Comment (Kameshwaran) [inline src/index.css]: Name should be `SoleilBold`

File: src/index.css [hunk 5]
```diff
@@ -64,6 +64,70 @@
...
+@font-face {
+  font-family: "SoleilBk";
+  font-weight: normal;
+  font-style: normal;
+  src: local("Soleil Bk"),
+  local("Soleil-Bk"),
+  url("./fonts/soleil-bk.otf") format("opentype");
+}
+@font-face {
+  font-family: "SoleilBk";
```
Comment (Kameshwaran) [inline src/index.css]: Name should be 'SoleilBkItalic'

File: src/index.css [hunk 6]
```diff
@@ -64,6 +64,70 @@
...
+  font-weight: normal;
+  font-style: italic;
+  src: local("Soleil Bk Italic"),
+  local("Soleil-Bk-Italic"),
+  url("./fonts/soleil-bk-italic.otf") format("opentype");
+}
+@font-face {
+  font-family: "SoleilSb";
```
Comment (Kameshwaran) [inline src/index.css]: Name should be 'SoleilSbItalic'

File: src/index.css (hunk 1 above)
Comment (ankushCb) [reply src/index.css]: Done.

File: src/index.css (hunk 3 above)
Comment (ankushCb) [reply src/index.css]: Done.

File: src/index.css (hunk 4 above)
Comment (ankushCb) [reply src/index.css]: Done.

File: src/index.css (hunk 5 above)
Comment (ankushCb) [reply src/index.css]: Done.

File: src/index.css (hunk 6 above)
Comment (ankushCb) [reply src/index.css]: Done.

File: src/index.css (hunk 2 above)
Comment (ankushCb) [reply src/index.css]: I think, this name is ok.
For Italic and bold alone we can change.
What do you think, Kamesh?

Comment (Kameshwaran) [reply src/index.css]: @ankushCb Not an issue.

**Pull request merged**